api/types/container/container.proto

```diff
@@ -3,10 +3,50 @@ syntax = "proto3";
 package containerd.v1.types;
 
 import "gogoproto/gogo.proto";
-import "github.com/docker/containerd/api/types/state/state.proto";
+
+enum Status {
+	CREATED = 0;
+	RUNNING = 1;
+	STOPPED = 2;
+	PAUSED = 3;
+}
 
 message Container {
 	string id = 1 [(gogoproto.customname) = "ID"];
-	string bundle = 2;
-	State state = 4;
+	uint32 pid = 2;
+	Status status = 3;
+}
+
+message Process {
+	uint32 pid = 1;
+	repeated string args = 2;
+	repeated string env = 3;
+	User user = 4;
+	string cwd = 5;
+	bool terminal = 6;
+	uint32 exit_status = 7;
+	Status status = 8;
+}
+
+message User {
+	uint32 uid = 1;
+	uint32 gid = 2;
+	repeated uint32 additional_gids = 3;
+}
+
+message Event {
```

Comment:
Let's namespace this as `ContainerEvent`, since they are in a common package with the rest of the types.